src/Service/GitlabService.php

```diff
@@ -70,6 +72,23 @@ public function notifyAboutReadyReviews(Author $author, Review $review): void
         }
     }
 
+    public function notifyAboutInitialReviewRequest(MergeRequest $mergeRequest, Scope $scope): void
+    {
+        $project = $mergeRequest->getProject();
+
+        $pattern = sprintf('%s(%s.*)$%si', $scope->getName(), chr(1), chr(1));
+        preg_match($pattern, $mergeRequest->getDescription(), $matches);
```

Comment:
I think it would be better to keep things simple and not use a preg_match.
If the scope is found using stripos and existing database of possible tags - then we create a review. If not - we just do nothing.

If we use the implementation I suggested, we can also reuse existing gitlab service methods. In that case, we can delete this new method altogether